projects/rawkode-academy/web/src/components/auth/profile.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import { initFlowbite } from 'flowbite';
+import { onMounted } from 'vue';
+
+onMounted(() => {
+	initFlowbite();
```

Comment:
Hmm, is https://github.com/RawkodeAcademy/RawkodeAcademy/blob/main/projects/rawkode-academy/web/src/wrappers/page.astro#L21 not enough or is Vue special here and needs a second round of init?

Reply:
Ah nevermind, saw the removal of said piece of code